resources/views/tickets/ignored/index.blade.php

```diff
@@ -0,0 +1,21 @@
+@extends('helpdesk::layout.main')
+
+@section('content')
+  <div class="section">
+    <div class="container">
+      <h1 class="title">Ignored Tickets</h1>
+
+      {{ $ignored->links('helpdesk::pagination.bulma') }}
+
+      @if (isset(auth()->user()->agent->is_super) && auth()->user()->agent->is_super)
+        @include('helpdesk::tickets.ignored.index.table-super', [
+          'tickets' => $ignored
+        ])
+      @else
+        <p>Nothing to see here.</p>
+      @endif
+
+      {{ $ignored->links('helpdesk::pagination.bulma') }}
+    </div>
+  </div>
+@endsection
```

Comment:
The way this entire thing works pops out at me. If an agent tries to get hold of a non-existent ticket, they will get a 404. But here they get a message that there's "nothing to see here". 

So for an agent, the difference between a ticket that doesn't exist and an ignored ticket isn't seamless -- there's some information to be gleaned in the difference between the responses.

You want to let the controller handle this instead, and abort with a 404 (or whatever happens for non-existent tickets, probably should be a 404).